supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -128,6 +128,29 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
     (directoryTransferBundle :: (filesWithSameNamesTransferBundles ++ filesWithDifferentNamesTransferBundles)) mkString "\n\n"
   }
 
+  private def gcsEgressCheckBundle[T <: PipelinesApiInput](transferConfiguration: GcsTransferConfiguration)(bucket: String, inputs: NonEmptyList[T]): String = {
+    val project = inputs.head.cloudPath.asInstanceOf[GcsPath].projectId
+    val maxAttempts = transferConfiguration.transferAttempts
+    // Use a digest as bucket names can contain characters that are not legal in bash identifiers.
+    val arrayIdentifier = s"egress_check_" + DigestUtils.md5Hex(bucket)
+    val entries = inputs.toList.map(_.cloudPath) mkString("\"", "\"\n|  \"", "\"")
+    
+    val localizationEgressValue = jobDescriptor.workflowDescriptor.workflowOptions.getOrElse(
+      WorkflowOptionKeys.LocalizationEgress, "local")
+    val localizationEgressStrictFlag = jobDescriptor.workflowDescriptor.workflowOptions.getBoolean(
+      WorkflowOptionKeys.LocalizationEgressStrict).getOrElse(false)
```

Comment:
both of these are available in the superclass `cromwell.backend.google.pipelines.common.PipelinesApiAsyncBackendJobExecutionActor`